distro/pom.xml

```diff
@@ -47,8 +47,8 @@
     <htmlwidgets.version>1.11.0</htmlwidgets.version>
     <serialization-xstream.version>0.2.16</serialization-xstream.version>
     <ordertemplates.version>1.0.2</ordertemplates.version>
-    <patientflags.version>3.0.6-SNAPSHOT</patientflags.version>
-    <o3forms.version>1.1.0-SNAPSHOT</o3forms.version>
+    <patientflags.version>3.0.5</patientflags.version>
+    <o3forms.version>1.1.0</o3forms.version>
```

Comment:
@Ruhanga , please update the o3forms version here.
Thanks!

Reply:
When Undoing `o3forms.version` update it to 1.2.0-SNAPSHOT. Otherwise LGTM